.buildkite/shared-pipeline-vars

```diff
@@ -6,4 +6,10 @@
 # 🎗️ If you update the image to a newer Xcode version, don't forget to also update the badge in the README.md file accordingly for consistency
 export IMAGE_ID="xcode-15.4"
 
+# Uncomment this to experiment with an Xcode beta only on PRs and TestFlight beta builds
```

Comment:
FYI: I wrote this intentionally as "Uncomment this" while in practice it's currently already uncommented, because my idea was for y'all to just comment the 3 `if…fi` lines below back once you'll be done with experimenting the Xcode 16 beta.

In other words, basically once the final Xcode 16.0 will be out, you'd update the `export IMAGE_ID="xcode-16.0"` from line 7 above, then comment the `if…fi` lines below to remove them but still keep the syntax around for next time.
Then, if some months later in the future after you migrated all builds to the stable `16.0`, you wanted to start experimenting again with the next Xcode beta (e.g. say Xcode 16.1b3), you'd just have to un-comment those lines below again and update the `export IMAGE_ID="xcode-16.1-beta3"` on line 12… instead of having to remember (or dig in git history) how to do such conditional when experimenting with betas.